### DISCUSSION TITLE
Data Security

### DISCUSSION THREAD

**Discussion by LuckyHedgehog** [sql-bi/Bravo#754] [Q&A]
Is there any documentation on Bravo's data security, specifically about whether it sends data anywhere?
For example, when I connect to a model, does the data from the model go anywhere other than the local machine where Bravo is running?

**Comment by marcosqlbi** [comment]
The short answer is that Bravo does not collect any data from the model: it only sends (if enabled) telemetry about which features are used in the tool and, in case of an application error, what feature raised the exception. The telemetry is sent to Application Insights, a Microsoft Azure service. However, if you disable the telemetry, Bravo does not send any data. When the telemetry is enabled, the telemetry does not include any personal data or any data of the models you analyze.

The long answer is that Bravo has several features that allow the user to connect with Power BI Service and interact with the service, potentially extracting data (like Export and Analyze model) and also sending data to the cloud (for example, you could save a VPAX file to OneDrive or export data to a cloud service or database). However, in these cases, you interact with services you access with your own account, and as explained before, Bravo does not collect (on telemetry) any data you access.
However, the best way to perform a security review is to look at the open-source code. You can inspect in detail what the telemetry does, or you can disable it and block any communication port with a firewall to prevent any possible communication.

I hope it helps!
